core/src/internal/franchise/player/player.service.ts

```diff
@@ -424,4 +424,26 @@ export class PlayerService {
             };
         }
     }
+
+    async playerCanSaveDemos(playerId: number): Promise<boolean> {
+        const player = await this.playerRepository.getById(playerId, {
+            relations: {
+                member: {
+                    platformAccounts: true,
```

Comment:
Is there a way to filter these into the correct game automatically? Feels like the steps beneath could be done at the database to do less work.

Reply:
#485f8e1